README.md

```diff
@@ -1,5 +1,5 @@
 # patreon-php
-Interact with the Patreon API via OAuth.
+Interact with the Patreon API via OAuth. (This is a development package that will be removed later. Please do not use this)
```

Comment:
I don't think we need this comment?

Reply:
Removed: https://github.com/Patreon/patreon-php/pull/58/commits/e580c0b60ca788c40aa42e01c464617b80b0faaf